packages/widgets-manager/src/utils/index.ts

```diff
@@ -0,0 +1,71 @@
+import { Position, HTMLFloatType } from './../types';
+
+/**
+ * Return an element by its id or throws an error if it can not find one
+ * @param id The dom element id
+ */
+const elementById = (id: string): HTMLElement => {
+  const element = document.getElementById(id);
+  if (element === null) throw Error('Can not find the dom element with id: ' + id);
+  return element;
+}
+
+/**
+ * Helper function to get an element bounding position 
```

Comment:
all functions help you do stuff, "helper function" makes no sense

`gets an element bounding position` is more than enough